controllers/api.controller.js

```diff
@@ -33,8 +34,20 @@ async function getAllArticles(req, res, next) {
   }
 }
 
+async function getCommentsOfArticle(req, res, next) {
+  const { article_id } = req.params;
+  try {
```

Comment:
You will need to check the reason for the query returning empty rows:
- article exists but has no comments.
- article does not exist.